lib/documents/document.js

```diff
@@ -28,6 +28,8 @@ export class Document extends BaseDocument {
   // contentKey: ID used to identify editor's content
```

Comment:
You have to rename this.

Reply:
Do you mean that I have to rename the `contentKey` field or that I have to change the field description?